### PR TITLE
Fix and simplify build instructions for Windows

### PR DESCRIPTION
**Pull request by adamretter**
Closes https://github.com/theia-ide/theia/issues/1614
Also relevant https://github.com/theia-ide/theia/issues/2009

### CODE REVIEW

**Comment by kittaakos** [comment]
I am reviewing it now...

> Also relevant #2009

Why is it relevant?

@adamretter, please sign your commit. Otherwise, we cannot accept it.

File: doc/Developing.md [hunk 1]
```diff
@@ -235,33 +235,61 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
 
 ## Building on Windows
 
-Run cmd.exe as an administrator and install `choco` by copy-pasting the command
-to your console:
+1. Run cmd.exe as an Administrator. You can do this by right-clicking "Command Prompt" in the Windows Start Menu, and then right-clicking "Run as Administrator".
```
Comment (kittaakos) [inline doc/Developing.md]: > You can do this by right-clicking "Command Prompt" in the Windows Start Menu

I must be blind, where is the "Command Prompt'?

screen shot 2018-08-31 at 10 03 00

File: doc/Developing.md [hunk 2]
```diff
@@ -235,33 +235,61 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
...
+```ps
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+3. Refresh the environment settings by running the following command in the Command Prompt: 
+```ps
+refreshenv
+```
+
+4. Install NodeJS via. choco (**NOTE**: the version is important!) by running the following in the Command Prompt:
```
Comment (kittaakos) [inline doc/Developing.md]: `NodeJS` -> `Node.js`

File: doc/Developing.md [hunk 3]
```diff
@@ -235,33 +235,61 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
...
+```ps
+choco install nodejs.install --version 8.11.4 -y
+```
+
+5. Install Yarn via. choco (**NOTE**: the version is important!) by running the following in the Command Prompt:
+```ps
+choco install yarn --version 1.7.0 -y
+```
+
+6. Install Git via. Choco:
```
Comment (kittaakos) [inline doc/Developing.md]: Choco -> choco (or `choco`).

Please if you escape something or put it between the double quotes, use it consistently.
"Command Prompt", Command Prompt
Choco, choco, `choco`.

File: doc/Developing.md [hunk 4]
```diff
@@ -235,33 +235,61 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
...
-Node.js and npm too:
-
-    choco install yarn
-
-Install `git` via `choco`
-
-    choco install git
-
-Install the correct version of `yarn` (The version is important)
+2. Install `choco` by copy and pasting the command below into the Command Prompt, and then pressing the `return` key to run it:
```
Comment (kittaakos) [inline doc/Developing.md]:
```
`return`
```
`return`

```
<kbd>return</kbd>
```
<kbd>return</kbd>

**Comment by adamretter** [comment]
#2009 is relevant because it states that Theia does not run on Node v10.3.0. This PR ensures that Node.js 8.11.4 is used.

File: doc/Developing.md (hunk 1 above)
Comment (adamretter) [reply doc/Developing.md]: Which version of Windows do you have? Mine looks significantly different to that...

Comment (kittaakos) [reply doc/Developing.md]: Windows 10 Pro.

**Comment by kittaakos** [comment]
> #2009 is relevant because it states that Theia does not run on Node v10.3.0. This PR ensures that Node.js 8.11.4 is used.

I see. I thought you're considering this PR as a fix of #2009.

**Comment by adamretter** [comment]
@kittaakos Okay, I made the changes you asked for, rebased and signed the commits.

File: doc/Developing.md (hunk 1 above)
Comment (kittaakos) [reply doc/Developing.md]: 😄 I was confused. I did the right click on the Start Menu. Great, then it is OK.
By the way there is another way without clicking: https://superuser.com/questions/968214/open-cmd-as-admin-with-windowsr-shortcut

 - <kbd>Win</kbd>+<kbd>R</kbd>
 - Type: `cmd`.
 - <kbd>Alt</kbd>+<kbd>Ctrl</kbd>+<kbd>Enter</kbd>

Comment (dunnry) [reply doc/Developing.md]: There is one small mistake here in the choco command for yarn.  You should use the -i option to ignore dependencies or you will find that yarn will install Node 10+ as well, overriding your initial Node LTS install.

**Comment by dunnry** [comment]
I would update the instructions to include the -i flag when installing yarn from chocolatey.  This tells chocolatey to not install node again.  If you forget this flag, it will likely pull in Node 10+ as well.

**Comment by adamretter** [comment]
@kittaakos okay I fixed the run as Administrator thing and squashed and signed the commits.

**Comment by kittaakos** [comment]
For committers/reviewers:
```
There is a valid ECA on file for adam.retter@googlemail.com.
```

✅

**Comment by vince-fugnitto** [comment]
@adamretter do you still plan on amending the PR in order for it to be merged?
I understand that the PR has been in review for quite a while now and perhaps it is outdated (there are conflicts at the moment).

**Comment by vince-fugnitto** [comment]
I'll be closing the PR for the moment due to inactivity :(
@adamretter please re-open the PR if you ever get a chance to update and resolve the issues, we'd love to have better and simpler documentation regarding build instructions for Windows.

**Pull request closed without merging**